pallets/utxo/src/lib.rs

```diff
@@ -401,7 +536,7 @@ pub mod pallet {
             .ok_or("Sub underflow")
             .unwrap();
 
-        log::debug!("disperse_reward:: reward total: {:?}", remainder);
+        log::debug!("reward_tx_validation:: reward total: {:?}", remainder);
         <RewardTotal<T>>::put(remainder as Value);
```

Comment:
As per the explanation above, if I understand how this works, this should never be happen, and the remainder should *always* be zero.